layouts/partials/primary_top.html

```diff
@@ -104,6 +104,26 @@ <h1 id="title">
                                         <span>&nbsp;</span>
                                         {{- printf (i18n "page_reading_time") .Page.ReadingTime -}}
                                     </span>
+
+                                    <span>&nbsp;</span>
+
+                                    <span>
+                                        {{if isset .Params "test"}}
+                                            {{ if(.Params.test) }}
```

Comment:
There a 3 possible values of .Params.test:
1. "yes"
2. "no"
3. "n/a"

1 should display green check, 2 should display grey check, 3 should display nothing (same as not isset).

Reply:
Ah sorry, The isset check around the code makes sure it is not displayed if "n/a"